atc/exec/artifact_output_step_test.go

```diff
@@ -59,11 +66,11 @@ var _ = Describe("ArtifactOutputStep", func() {
 		})
 	})
 
-	Context("when the source exists", func() {
+	Context("when the artifact exists", func() {
 		Context("when the source is not a worker.Volume", func() {
```

Comment:
guessing we need to update these test descriptor strings: `source` -> `artifact`